docs/containers-transports.5.md

```diff
@@ -43,6 +43,7 @@ The containers-registries.conf(5) further allows for configuring various setting
 Note that a _docker-reference_ has the following format: `name[:tag|@digest]`.
 While the docker transport does not support both a tag and a digest at the same time some formats like containers-storage do.
 Digests can also be used in an image destination as long as the manifest matches the provided digest.
+The docker transport supports pushing images without a tag or digest to a registry when the image name is suffixed with `@@unknown-digest@@`. Images without a tag or digest can be used to create manifest lists after the images have been pushed.
 The digest of images can be explored with skopeo-inspect(1).
```

Comment:
… or, if the destination uses the **@unknown-digest@** syntax, the writer of the manifest must provide it some other way.

Reply:
Do you want me to add this as well to the docs? I fixed up the docs a bit, let me know if you want this added as well.

Reply:
I would weakly prefer this to be documented _somehow_, because I think it’s not immediately obvious how to use `@@unknown-digest@@` (or destinations with a specified digest).

OTOH doing that clearly, and without misleading most users who don’t need to deal with this, seems hard to do.

Meh, we can assume the current text is good enough (at least good enough to find this PR :) ) and we can improve it based on user feedback or proposed PRs.

Reply:
Sounds good, I am also not sure how to add this clearly. Will update based on any user feedback in the future.